### PR TITLE
Sync will now only list unread messages.

### PR DESCRIPTION
**Pull request by megatrond**

### CODE REVIEW

**Comment by asbjornenge** [comment]
Hm... This might be a good idea a little further down the road. Right now I like to see what I sync (for testing mostly). But I'll keep it and we can rebase and merge when the time comes.

Thanks!

File: index.js [hunk 1]
```diff
@@ -19,8 +19,24 @@ keytalk.prototype.read_config = function(callback) {
...
+    
     p.stdout.on('data', function(data) {
-        if (typeof callback == 'function') callback(data)
+        //if (typeof callback == 'function') callback(data)
+        returnData = data;
+
+    })
+    p.on('exit', function(code) {
+        // what can code be, and when is it an error?
+        console.log('Keybase exit code ' + code);
```
Comment (megatrond) [inline index.js:36]: This is just for testing, in case keybase returns some other exit code

Comment (asbjornenge) [inline index.js:36]: NP! I'll fix that bit...

**Comment by megatrond** [comment]
Closing this because its not really needed.

**Pull request closed without merging**